gin.go

```diff
@@ -311,6 +312,16 @@ func (engine *Engine) Run(addr ...string) (err error) {
 	return
 }
 
+// RunFCGI attaches the router to a fcgi.child and starts listening to os.Stdin for requests
+// It is a shortcut for fcgi.Serve(nil, router)
+func (engine *Engine) RunFCGI() (err error) {
+	defer func() { debugPrintError(err) }()
+
+	debugPrint("Listening and serving HTTP on FCGI\n")
+	err = fcgi.Serve(nil, engine)
+	return
+}
+
```

Comment:
Please add the unit test.

Reply:
Ok. Quick question for you though: if I do things correctly, will you merge the modification and push the fact that `gin` now will be compiled heavier than before, even if no one seems to use it? I wonder if there would be a better way to do it, like update just the documentation to describe how to use `gin` with `fcgi` without modifications. For example:
`err := fcgi.Serve(nil, router)`